.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "verification/src/proto/googleapis"]
+	path = verification/src/proto/googleapis
+	url = https://fuchsia.googlesource.com/third_party/googleapis
```

Comment:
Smh it doesn't work without it:
https://stackoverflow.com/questions/66168350/import-google-api-annotations-proto-was-not-found-or-had-errors-how-do-i-add